### DISCUSSION TITLE
PT-BR Documentation 🇧🇷

### DISCUSSION THREAD

**Discussion by abiliocoelho** [adonisjs/core#2342] [Ideas]
Maybe the community could help with a translation for documentation for PT-BR, in the discord group, several people became interested.

**Comment by iagobruno** [comment]
I'm available to help translate the documentation.

**Reply by arthurcvm** [reply]
Me too

**Reply by thetutlage** [reply]
Please follow the README of the docs repo to get started with the translations https://github.com/adonisjs/docs.adonisjs.com

https://github.com/adonisjs/docs.adonisjs.com/issues/1

**Reply by guntribam** [reply]
Please look into this [issue](https://github.com/adonisjs/docs.adonisjs.com/issues/8)
@arthurcvm @iagobruno

**Reply by guntribam** [reply]
@tavaresgerson @abiliocoelho @diegogit03

**Reply by tavaresgerson** [reply]
@guntribam Starting... :-)

**Comment by tavaresgerson** [comment]
Hi! This is done for 4.1 version https://github.com/tavaresgerson/adonisdocbr :smile:

**Reply by abiliocoelho** [reply]
Thank you for sharing. I will share in the community. 🇧🇷

**Comment by diegogit03** [comment]
and how we can move this doc to official doc?
